### DISCUSSION TITLE
HLTV DIR

### DISCUSSION THREAD

**Discussion by gameinside** [SmileYzn/PugMod-AMXX#36] [General]
It would be necessary to change a little PUGHLTV files are recorded in the cstrike directory, I would advise you to make cstrike/demos or cstrike/hltv there at your discretion, why exactly, as for me it is more convenient

**Comment by SmileYzn** [comment]
You Just need to put prefix with that folder 👍 

Like demos/demoname.
I think it need to create folder manually
